src/components/amino-acid-slider.js

```diff
@@ -139,10 +139,6 @@ class AminoAcidSlider extends Component {
       <rect x={acidWidth/2 + (loc * (acidWidth * 1.1)) - 1} y="1" width="19" height="20" style={{fill: "#33F", stroke: "#AAF", strokeWidth: 2}} />
```

Comment:
Not sure why I wasn't seeing this before, but I'm getting a duplicate key warning about the marks here and in `Protein.js` now once there are any marks.

Reply:
Fixed. Yeah, odd that that warning wasn't showing in master.